.eslintrc.js

```diff
@@ -1,31 +1,255 @@
+    'no-debugger': 'error',
+    'no-delete-var': ['error'],
+    'no-dupe-args': ['error'],
+    'no-dupe-class-members': 'warn',
+    'no-dupe-keys': ['error'],
+    'no-duplicate-case': ['error'],
+    'no-empty': [
       'error',
       {
-        arrays: 'always-multiline',
-        objects: 'always-multiline',
-        imports: 'always-multiline',
-        exports: 'always-multiline',
-        functions: 'never',
+        allowEmptyCatch: true,
+      },
+    ],
+    'no-empty-character-class': ['error'],
+    'no-empty-pattern': ['error'],
+    'no-ex-assign': ['error'],
+    'no-extra-boolean-cast': ['error'],
+    'no-extra-semi': 'off',
+    'no-fallthrough': 'off',
+    'no-func-assign': ['error'],
+    'no-global-assign': ['error'],
+    'no-inner-declarations': ['error'],
+    'no-invalid-regexp': ['error'],
+    'no-irregular-whitespace': 'off',
+    'no-misleading-character-class': ['error'],
+    'no-mixed-spaces-and-tabs': ['error'],
+    'no-new-symbol': ['error'],
+    'no-obj-calls': ['error'],
+    'no-octal': ['error'],
+    'no-prototype-builtins': ['error'],
+    'no-redeclare': 'warn',
+    'no-regex-spaces': ['error'],
+    'no-self-assign': ['error'],
+    'no-shadow-restricted-names': ['error'],
+    'no-sparse-arrays': ['error'],
+    'no-this-before-super': ['error'],
+    'no-undef': ['error'],
+    'no-unexpected-multiline': ['error'],
+    'no-unreachable': ['error'],
+    'no-unsafe-finally': 'error',
+    'no-unsafe-negation': ['error'],
+    'no-unused-labels': 'error',
+    'no-useless-catch': ['error'],
+    'no-useless-escape': 'warn',
+    'no-with': ['error'],
+    'require-yield': ['error'],
+    'use-isnan': 'error',
+    'valid-typeof': 'off',
+    '@typescript-eslint/adjacent-overload-signatures': 'error',
+    '@typescript-eslint/array-type': [
+      'warn',
+      {
+        default: 'array',
+      },
+    ],
+    '@typescript-eslint/ban-types': [
+      'error',
+      {
+        types: {
+          Object: {
+            message: 'Avoid using the `Object` type. Did you mean `object`?',
+          },
+          Function: {
+            message: 'Avoid using the `Function` type. Prefer a specific function type, like `() => void`.',
+          },
+          Boolean: {
+            message: 'Avoid using the `Boolean` type. Did you mean `boolean`?',
+          },
+          Number: {
+            message: 'Avoid using the `Number` type. Did you mean `number`?',
+          },
+          String: {
+            message: 'Avoid using the `String` type. Did you mean `string`?',
+          },
+          Symbol: {
+            message: 'Avoid using the `Symbol` type. Did you mean `symbol`?',
+          },
+        },
+      },
+    ],
+    '@typescript-eslint/class-name-casing': 'error',
+    '@typescript-eslint/prefer-regexp-exec': 'warn',
+    '@typescript-eslint/prefer-string-starts-ends-with': 'warn',
+    '@typescript-eslint/unbound-method': 'warn',
+    '@typescript-eslint/no-unused-vars': 'warn',
```

Comment:
this rule we need to change. With `tslint`, it is an error. So here we need to configure it as`error` as well and configure https://eslint.org/docs/rules/no-unused-vars#argsignorepattern to ignore variable names which start with `_`

Reply:
This will most likely be fixed by the incoming commit